tests/bagatom/test_bagToXML.py

```diff
@@ -0,0 +1,208 @@
+import os
+import pytest
+
+from mock import mock_open
+
+from codalib import bagatom
+
+
+# Settings for the monkeypatched fixtures
+TEST_NAME = 'baz'
+TEST_PATH = os.path.join('/foo/bar', TEST_NAME)
+FILE_COUNT = 3
+PAYLOAD_SIZE = 1500
+BAGGING_DATE = '2015-01-01'
+BAGIT_CONTENTS = 'BagIt-Version: 0.96\nTag-File-Character-Encoding: UTF-8'
+
+
+@pytest.fixture
+def bagxml(monkeypatch):
+    """
+    Test fixture that patches the following functions.
+        open()
+        bagatom.getBagTags()
+        bagatom.getOxum()
+    """
+    m = mock_open(read_data=BAGIT_CONTENTS)
+    monkeypatch.setattr('__builtin__.open', m)
+
+    monkeypatch.setattr('codalib.bagatom.getBagTags', lambda x: {})
+
+    monkeypatch.setattr(
+        'codalib.bagatom.getOxum',
+        lambda x: '{0}.{1}'.format(PAYLOAD_SIZE, FILE_COUNT))
```

Comment:
Actually, after discussing that with @damonkelley, I thought I was simply imposing personal preference and was going to withdraw the suggestion. However, upon looking at PEP8, I am now confused. Look towards the bottom of the [Indentation](https://www.python.org/dev/peps/pep-0008/#indentation) to see examples of what I mean. From _my_ interpretation, it seems that if you do not include an argument on the first line of the function call, then you must align the closing parenthesis with the first character of the function name or with the first character of the line above it. I may (very likely) be interpreting it wrong though.